underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -586,6 +599,22 @@ private StoredObject getObject(final String path) {
     return container.getObject(stripContainerPrefixIfPresent(path));
   }
 
+  /**
+   * Delete an object if it exists.
+   *
+   * @param object object handle to delete
+   * @return true if object deletion was successful
+   */
+  private boolean deleteObject(final StoredObject object) {
+    try {
+      object.delete();
+      return true;
+    } catch (NotFoundException e) {
+      LOG.error("Object not found {}", e);
```

Comment:
NIT: is this intentionally printing out the `NotFoundException` and not the `object`?